extensions/Themadpunter/rsa.js

```diff
@@ -0,0 +1,89 @@
+// Name: RSA
+// ID: themadpunter-rsa
+// Description: Encrypt text and files using RSA, the world's best asymmetric encryption algorithm.
+// By: Themadpunter <https://scratch.mit.edu/users/The_Mad_Punter/>
+// License: MIT
+
+(function(Scratch) {
+    'use strict';
+
+    // Load the JSEncrypt library from a CDN
+    const script = document.createElement('script');
+    script.src = 'https://cdn.jsdelivr.net/npm/jsencrypt/bin/jsencrypt.min.js';
+    document.head.appendChild(script);
```

Comment:
You shouldnt be fetching a file url like that as:
- this forces people to need internet for this extension to work
- url could go down at any point

You should use a data.uri instead.
Also add `/* global [library name here] */` to fix the lint errors

Reply:
the desktop app still needs to work without internet (this is why the extension library there can only update when the app does, because it's stored offline).
for importing, i think using `await import()` and making the extension function async is a simple option, but you should still make the extension url a data url because of the offline part